share/completions/quilt.fish

```diff
@@ -1,35 +1,35 @@
 # completions for quilt version 0.46 or similar
-complete -c quilt -s h -d "show help"
-complete -c quilt -l version -d "show version"
-complete -c quilt -l quiltrc -r --no-file -d "specify the file to Run Control file to use"
+complete -c quilt -s h -d "Show help"
+complete -c quilt -l version -d "Show version"
+complete -c quilt -l quiltrc -r --no-file -d "Specify the file to Run Control file to use"
 complete -c quilt -l trace -r --no-file -d "Runs the command in bash trace mode (-x). For internal debugging"
 
-complete -c quilt -a add -r -d "Add one or more files to the topmost or named patch"
-complete -c quilt -a annotate -r -d "Print an annotated listing of the specified file"
+complete -c quilt -a add -r -d "Add file(s) to the topmost/named patch"
+complete -c quilt -a annotate -r -d "Print an annotated listing of the given file"
 complete -c quilt -a applied -r -d "Print a list of applied patches"
-complete -c quilt -a delete -r -d "Remove the specified or topmost patch from the series file"
-complete -c quilt -a diff -r -d "Produces a diff of the specified file(s) in the topmost  or  specified patch"
-complete -c quilt -a edit -r -d "Edit the specified file(s) in $EDITOR"
-complete -c quilt -a files -r -d "Print the list of files that the topmost or specified patch changes"
-complete -c quilt -a fold -r -d "Integrate the patch read  from  standard  input  into  the  topmost patch"
-complete -c quilt -a fork -r --no-file -d " Fork the topmost patch"
-complete -c quilt -a graph -r --no-file -d "Generate  a  dot(1) directed graph showing the dependencies between applied patches"
-complete -c quilt -a grep -r --no-file -d "Grep through the source files, recursively, skipping patches and quilt meta-information"
-complete -c quilt -a header -r --no-file -d " Print or change the header of the topmost or specified patch"
+complete -c quilt -a delete -r -d "Remove the topmost/given patch from the series file"
+complete -c quilt -a diff -r -d "Produces a diff of the given file(s) in the topmost/given patch"
+complete -c quilt -a edit -r -d "Edit the given file(s) in $EDITOR"
+complete -c quilt -a files -r -d "Lists files that the topmost/given patch changes"
+complete -c quilt -a fold -r -d "Integrate the patch read from standard input into the topmost patch"
+complete -c quilt -a fork -r --no-file -d "Fork the topmost patch"
+complete -c quilt -a graph -r --no-file -d "Generates a dot(1) directed graph of dependencies between applied patches"
+complete -c quilt -a grep -r --no-file -d "Grep through source files recursively (skips patches and quilt meta-information)"
```

Comment:
We already know that it's source files, no need to list patches and meta-stuff specifically. So just "grep through source recursively"?